openapi-spec/swagger.yaml

```diff
@@ -532,11 +516,156 @@ paths:
       summary: Delete a policy.
       operationId: deletePolicy
       responses:
+        '200':
+          description: Delete success
         '400':
           description: Invalid ID supplied
+  '/v1/s3/{bucketName}':
+    parameters:
+      - name: bucketName
+        in: path
+        description: the name of bucket
+        required: true
+        schema:
+          type: string
+    put:
+      tags:
+        - s3
+      summary: create a bucket.
+      operationId: bucketPut
+      responses:
         '200':
-          description: Delete success
-          
+          description: successful create bucket
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/BaseResponse'
+      requestBody:
+        content:
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/CreateBucketConfiguration'
+    delete:
+      tags:
+        - s3
+      summary: delete a bucket.
+      operationId: bucketdelete
+      responses:
+        '200':
+          description: successful delete bucket
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/BaseResponse'
+    get:
+      tags:
+        - s3
+      summary: list objects
+      operationId: listObjects
+      responses:
+        '200':
+          description: successful list objects
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ListObjectResponse'
+  /v1/s3/:
+    parameters:
+      - name: bucketName
+        in: path
+        description: the name of bucket
+        required: true
+        schema:
+          type: string
+    get:
+      tags:
+        - s3
+      summary: list buckets.
+      operationId: listBuckets
+      responses:
+        '200':
+          description: successful delete bucket
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ListAllMyBucketsResult'
+  '/v1/s3/{bucketName}/{Object}':
+    parameters:
+      - name: bucketName
+        in: path
+        description: the name of bucket
+        required: true
+        schema:
+          type: string
+      - name: Object
+        in: path
+        description: the name of object
+        required: true
+        schema:
+          type: string
+      - name: Range
+        in: query
+        description: >-
+          the Header parameter Range value like "bytes:1-10",used for multipart
+          download
+        required: false
+        schema:
+          type: string
+      - name: uploads
+        in: query
+        description: 'the format is ''/{object}?uploads'',used for init multipartupload'
+        required: false
+        schema:
+          type: string
+      - name: uploadId&partNumber
```

Comment:
`uploadId&partNumber` is not a suitable expression.

Reply:
it has been changed,now it is right